docs/MoreAboutPersistentStorage.md

```diff
@@ -0,0 +1,44 @@
+# Persistent Stores used in ACM Observability
+
+ACM Observability is a stateful application. It creates following persistent volumes (there are more than 1 copies as it runs as stateful sets).
+
+### List of Stateful sets
+
+| Name | Purpose |
+| ----------- | ----------- |
+| alertmanager | Alertmanager stores its silencing, nflog etc in its storage. nflog is append-only log of active/resolved notifications along with the notified receiver, and a hash digest of the notification's identifying contents|
+| thanos-compact | The compactor needs local disk space to store intermediate data for its processing as well as bucket state cache. Generally, for medium sized bucket about 100GB should be enough to keep working as the compacted time ranges grow over time. However, this highly depends on size of the blocks. In worst case scenario compactor has to have space adequate to 2 times 2 weeks (if your maximum compaction level is 2 weeks) worth of smaller blocks to perform compaction. First, to download all of those source blocks, second to build on disk output of 2 week block composed of those smaller ones. On-disk data is safe to delete between restarts and should be the first attempt to get crash-looping compactors unstuck. However, it’s recommended to give the Compactor persistent disk in order to effectively use bucket state cache between restarts. |
+| thanos-rule | The thanos ruler evaluates Prometheus recording and alerting rules against chosen query API via repeated query. Rule results are written back to disk in the Prometheus 2.0 storage format. |
+| thanos-receive-default | Thanos receiver accepts incoming data (Prometheus remote-write requests) and writes these into a local instance of the Prometheus TSDB. Periodically (every 2 hours) TSDB blocks are uploaded to the Object storage for long term storage and compaction. The amount of hours/days of data retained in this stateful set (acts a local cache) was fixed in ACM 2.1/2.2. It has been exposed as an API parameter in ACM 2.3: _RetentionInLocal_ |
+| thanos-store-shard| It acts primarily as an API gateway and therefore does not need significant amounts of local disk space. It joins a Thanos cluster on startup and advertises the data it can access. It keeps a small amount of information about all remote blocks on local disk and keeps it in sync with the bucket. This data is generally safe to delete across restarts at the cost of increased startup times. |
+| thanos-store-memcached | Thanos Store Gateway supports an index cache using Memached to speed up postings and series lookups from TSDB blocks indexes. |
+| | |
+
+
+
+
+### Configuring the Stateful sets
+
+In ACM 2.1 and ACM 2.2 only the following was exposed using the Mutlicluster Observability Operator API - _observability.open-cluster-management.io/v1beta1_
+
+```
+    //defaults shown below
+    statefulSetSize: 10Gi
+    statefulSetStorageClass: gp2
+```
```

Comment:
Based on my previous comment about versioning, I recommend that we update this to read:

In previous versions of the `open-cluster-management` project, only the `statefulSetSize` and `statefulSetStorageClass` could be configured with the Multicluster Observability Operator API - `observability.open-cluster-management.io/v1beta1`

Instead of using italics for the API, use monospace